awesome.js

```diff
@@ -396,9 +623,19 @@ class Awesome{
 
         /**
          * uniqueEntries ensures that keys and values of data array are unique
+         *
+         * @example
+         *
+         * let arr = ['green', 'red', 'white', 'black', 'red'];
+         *
+         * awesome.uniqueEntries(arr);
+         *
+         * //respose is
+         * 'duplicate value string of red found on 4 && 1 const value strings MUST be unique!'
+         *
```

Comment:
Please also add an object example as that is infact what this is intended to validate, hence unique keys, and how / where it is used in the code.

Interesting example though, I did not think to use this that way, but I guess it works for that too.